src/main/kotlin/racing/Infrastructure/RacingCar.kt

```diff
@@ -54,4 +53,17 @@ class RacingCar {
         }
         return false
     }
+
+    fun getWinner(result: List<Cars>): List<Winner> {
+        val sortedBy = sortedByDistanceDesc(result.last())
+        val longestDistance = longestDistance(sortedBy)
+
+        return sortedBy.filter { car -> car.distance.equals(longestDistance) }.map { car -> Winner(car.name) }
```

Comment:
이런식으로 고차함수를 넘기는 함수들은 보통 한 줄에 하나씩 호출합니다.

```suggestion
        return sortedBy
            .filter { car -> car.distance.equals(longestDistance) }
            .map { car -> Winner(car.name) }
```